assets/css/v2/dup/departures/time.scss

```diff
@@ -38,8 +38,19 @@
 }
 
 .departure-time__timestamp {
-  font-weight: 700;
-  font-size: 1em;
-  margin-right: 0.0694em;
-  line-height: 1.185em;
+  direction: rtl;
+}
+
+.departure-time__time {
+  font-weight: 800;
+  font-size: 138px;
+  line-height: 128px;
+}
+
+.departure-time__ampm {
+  margin-left: 20px;
+  font-weight: 400;
+  font-size: 96px;
+  line-height: 108px;
+  letter-spacing: 2px;
```

Comment:
Where did this CSS come from? I don't see this in the figma

Reply:
https://www.figma.com/file/xCNFkMDWpq7y7cIH4V5dln/DUP-screens?node-id=520-10597&t=JNmkxOCFTeH9rxOe-4

Reply:
I feel dumb! I panned around and never found this section 😅